CONTRIBUTING.md

```diff
@@ -194,6 +194,12 @@ Then you can connect to TiDB with mysql client.
 mysql -h127.0.0.1 -P4000 -uroot test
 ```
 
+If you use mysql client 8, you must be get error `ERROR 1105 (HY000): Unknown charset id 255`, you can add `--default-character-set utf8` in mysql client 8's arguments. [Issue #6942](https://github.com/pingcap/tidb/issues/6942)
```

Comment:
s/you must be get error/you may get the error/

Reply:
OK, I fixed.